contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -10045,6 +10045,18 @@ bool reset_search_path(PLtsql_stmt_execsql *stmt, char *old_search_path)
 	const char	*user = NULL;
 	const char	*schema;
 	top_es_entry = exec_state_call_stack->next;
+
+	if (stmt->func_call && stmt->schema_name != NULL && strncmp(stmt->schema_name, "sys", 3) != 0)
+	{
+		physical_schema = get_physical_schema_name(cur_dbname, stmt->schema_name);
+		dbo_schema = get_dbo_schema_name(cur_dbname);
+		new_search_path = psprintf("%s, %s, %s", physical_schema, dbo_schema, old_search_path);
+		/* Add the schema where the object is referenced and dbo schema to the new search path */
+		(void) set_config_option("search_path", new_search_path,
```

Comment:
do we need to reset it later?

Reply:
Yes, we should reset the search path if we make any changes to it otherwise it may affect the next statements.